package.json

```diff
@@ -13,7 +13,7 @@
     "check-types": "tsc -p tsconfig.json --noEmit",
     "clean": "rm -rf node_modules && rm -rf build && yarn install",
     "codegen": "graphql-codegen --config codegen.ts",
-    "cy:open": "cypress open",
+    "cy:open": "yarn evg-db-ops --reseed-and-dump; cypress open",
```

Comment:
I see this error consistently when running `yarn cy:open`, are you able to replicate? The tests fail to run locally as a result.

```
2023-09-06T11:55:04.911-0400	writing evergreen_local.parser_projects to /var/folders/11/qpgw1klx2zq5dwt86c952yl80000gp/T/evg_dump/evergreen_local/parser_projects.bson
2023-09-06T11:55:04.915-0400	Failed: error writing data for collection `evergreen_local.tasks` to disk: error reading collection: connection pool for localhost:27017 was cleared because another operation failed with: connection() error occurred during connection handshake: connection(localhost:27017[-13]) incomplete read of message header: read tcp 127.0.0.1:62565->127.0.0.1:27017: read: connection reset by peer
Error creating dump from evergreen_local db.
error Command failed with exit code 1.
```

Reply:
I'm getting the same thing

Reply:
Thanks for figuring this out! It would be good to document this somewhere like the new hire guide.